src/constants/dataTypes.js

```diff
@@ -0,0 +1,12 @@
+export const STREAM = {
+	ALL: 'STREAM.ALL',
```

Comment:
`ANY` would seem to be more appropriate than `ALL`. Thinking of search/scoping semantics, where `ALL` reduces rather than expands the set.

Reply:
Great thinking -- Also, I wish it could be *, but that would be annoying and break the . syntax for those cases.

Reply:
Yeah. Plus, being able to consistently access the constant name through dot notation instead of whatever you call `[]` notation is :chefkiss:.